app/src/main/java/com/greenfox/fuchsit/zerdareader/server/MockServer.java

```diff
@@ -33,15 +33,26 @@ public void enqueue(Callback<ArrayList<NewsItem>> callback) {
     }
 
     @Override
-    public MockCall<UserResponse> loginUser(String username, String password) {
+    public MockCall<UserResponse> loginUser(final String email, final String password) {
```

Comment:
@regnisalram actually this is not what the API spec says:
```
Expected input
{"email": "example@gmail.com", "password": "12345"}
```
Meaning it should be one object, lets call it `LoginRequest` for example